Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -121,22 +121,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Issue7865.xaml.cs">
       <SubType>Code</SubType>
     </Compile>
-    <Compile Include="$(MSBuildThisFileDirectory)Issue7803.xaml.cs">
-      <DependentUpon>Issue7803.xaml</DependentUpon>
-    </Compile>
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8087.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8222.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8167.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8366.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8269.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8529.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8529_1.xaml.cs">
-      <DependentUpon>Issue8529_1.xaml</DependentUpon>
-      <SubType>Code</SubType>
-    </Compile>
-    <Compile Include="$(MSBuildThisFileDirectory)Issue8508.xaml.cs">
-      <SubType>Code</SubType>
-    </Compile>
```

Comment:
Where did all these issues go? 😱

Reply:
Rebasing is fun

Reply:
But are they supposed to be gone here? Or you need to add them back?

Reply:
Lets act like they were never gone